can/io/generic.py

```diff
@@ -28,7 +28,7 @@ class BaseIOHandler(ContextManager, metaclass=ABCMeta):
 
     file: Optional[can.typechecking.FileLike]
 
-    def __init__(
+    def __init__(  # pylint: disable=unused-argument
```

Comment:
Is there a point in keeping `*args` and `**kwargs` if we don't use them? Don't they just hide mistyped arguments?

Reply:
I will try to remove all `*args`, i think they serve no purpose in this library. However, in most classes, CLI kwargs might get passed down, so i'd keep those even if they are unused.

Reply:
@zariiii9003 Sounds reasonable to me.

Reply:
Yeah, `kwargs` are much safer 👍